packages/SidePanel/src/components/Header/Header.js

```diff
@@ -31,24 +33,27 @@ function darkBackgroundProps(kind) {
 const Header = React.forwardRef((props, ref) => {
   const {
     hasCloseButton,
-    kind,
     isCompact,
+    kind,
+    level,
     // injected by Dialog.js
     onClose,
     ...moreProps
   } = props;
 
   return (
     <div data-pka-anchor="sidepanel.header" ref={ref} css={headerCSS} kind={kind} isCompact={isCompact} {...moreProps}>
-      <div tabIndex="-1">{props.children}</div>
+      <Heading tabIndex="-1" level={level || 2}>
+        {props.children}
+      </Heading>
```

Comment:
This is the breaking change, we enforce Heading now, consumer does not need to pass in <Heading> to the <SidePanel.Header> anymore.

Reply:
@mikrotron Do we have cases where we don't want to pass in Heading into the side panel title?

Reply:
Probably a better question for @johnpeele.  But _maybe_ we'd want to have the close button, but no actual heading...?